app/constants/endpoints.js

```diff
@@ -34,5 +34,8 @@ export default {
     MENTORSHIP_TICKET_FORM: '/v1/form/MentorshipTicket',
     ACCEPT_MENTORSHIP_TICKET: '/v1/mentorship/accept',
     UNACCEPT_MENTORSHIP_TICKET: '/v1/mentorship/unaccept',
-    COMPLETE_MENTORSHIP_TICKET: '/v1/mentorship/complete'
+    COMPLETE_MENTORSHIP_TICKET: '/v1/mentorship/complete',
+
```

Comment:
Another phantom space? I'm okay with dividing sections like this but could we apply them for all of the other constants?